cmd/tke-installer/app/installer/installer.go

```diff
@@ -455,6 +483,16 @@ func (t *TKE) initSteps() {
 		}...)
 	}
 
+	// TODO: when expansion enabled, auto set t.Para.Config.Application to be true
```

Comment:
after https://github.com/tkestack/tke/issues/1227,  t.Para.Config.Application is `true` by default